app/src/main/java/app/revanced/integrations/patches/components/LayoutComponentsFilter.java

```diff
@@ -136,6 +136,11 @@ public LayoutComponentsFilter() {
                 "cell_divider" // layout residue (gray line above the buttoned ad),
         );
 
+        final var chipsShelf = new StringFilterGroup(
+                SettingsEnum.HIDE_CHIPS_SHELF,
+                "chips_shelf"
```

Comment:
This looks like it could accidentally hide other "chips shelfs". It technically wouldn't be a problem as the description says "Hide chips shelf" so it applies to every, but its unintended and would be good to know.

Reply:
Should description be modified?

Reply:
No, but can you check if it filters any other chips shelf such as in the comment section or anywhere else?

Reply:
Comments chips, and expandable chips are shown without any side effects.